modules/ui.go

```diff
@@ -0,0 +1,207 @@
+package modules
+
+import (
+	"embed"
+	"strings"
+
+	"github.com/GoMudEngine/GoMud/internal/events"
+	"github.com/GoMudEngine/GoMud/internal/plugins"
+	"github.com/GoMudEngine/GoMud/internal/rooms"
+	"github.com/GoMudEngine/GoMud/internal/users"
+	"github.com/GoMudEngine/GoMud/internal/mudlog"
+)
+
+var (
+	//go:embed ui/*
+	ui_Files embed.FS
+	
+	// Instance accessible from elsewhere in this package
+	uiModule UIModule
```

Comment:
If this isn't used anywhere, is it needed? (Assuming my function receiver comment solves this)